lighthouse-core/test/results/sample_v2.json

```diff
@@ -3787,41 +4202,18 @@
       "id": "unminified-javascript",
       "title": "Minify JavaScript",
       "description": "Minifying JavaScript files can reduce payload sizes and script parse time. [Learn more](https://web.dev/unminified-javascript/).",
-      "score": 0.88,
+      "score": 1,
```

Comment:
is the polyfill actually minified or did we have a regression here?

Reply:
we got rid of zone.js

lighthouse-cli/test/fixtures/dobetterweb/third_party/aggressive-promise-polyfill.js is minified. shall we un minify it